EliteDangerous/Properties/Resources.resx

```diff
@@ -673,4 +673,7 @@
   <data name="starcluster" type="System.Resources.ResXFileRef, System.Windows.Forms">
     <value>..\Resources\starcluster.PNG;System.Drawing.Bitmap, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</value>
   </data>
+  <data name="estval" type="System.Resources.ResXFileRef, System.Windows.Forms">
+    <value>..\Resources\estval.png;System.Drawing.Bitmap, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</value>
```

Comment:
Just a nitpick, since this is only used by the UI, shouldn't this be in the UI project?

Reply:
It should, IMO, but I was being consistent with the placement of the other popout control icons.